src/components/contact/Contact.js

```diff
@@ -6,14 +6,18 @@ import styles from './assets/contact.scss';
 import peopleSearch from './assets/people-search.svg';
 import SocialMedia from './SocialMedia';
 
+export const EMAIL = 'codingcoachio@gmail.com';
```

Comment:
Let's create a configuration file, where we get the value from the environment or assign a default value if empty: `src/config/Constants.js`

```
export default {
  contact: {
    EMAIL:  process.env.EMAIL ||  'codingcoachio@gmail.com',
    SLACK_URL: process.env.SLACK_URL ||  '......',
  },
  social: {
    FB_URL: process.env.FACEBOOK_URL || 'https://www.facebook.com/codingcoachio/',
    INSTA_URL: process.env.INSTA_URL ||  'https://www.instagram.com/',
    TWITTER_URL: process.env.TWITTER_URL  || 'https://twitter.com/codingcoach_io',
  },
};
```

Then we will use the config object with the values from the env or the default.